packages/edit-post/src/components/header/header-toolbar/index.js

```diff
@@ -136,6 +136,7 @@ function HeaderToolbar( { setListViewToggleElement } ) {
 			className="edit-post-header-toolbar"
 			aria-label={ toolbarAriaLabel }
 			shouldUseKeyboardFocusShortcut={ ! blockToolbarCanBeFocused }
+			variant={ 'unstyled' }
```

Comment:
When working with strings, it's more straightforward/readable to just pass it in the string attribute value style:

```suggestion
			variant="unstyled"
```

otherwise, the difference is very minor and boils down to a `JSXExpressionContainer` vs `JSXIdentifier` under the hood, where one can further be evaluated and the other is already evaluated.

Reply:
Good catch! You've shared this with me before, and so I actively try to remember, but this one got away. 🙂